include/limits.h

```diff
@@ -309,4 +309,9 @@
 
 #define HOST_NAME_MAX  32
 
+/* ptrdiff_t limits */
+
+#define PTRDIFF_MAX INT_MAX
```

Comment:
PTRDIFF_MAX isn't always equals to INT_MAX

Reply:
Oh.. you are right. I somehow looked it up wrongly. It is `_ssize_t`, which is defined by architecture. That makes it way more complicated.